src/config/config.ts

```diff
@@ -0,0 +1,55 @@
+import fs from "fs";
+import path from "path";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+export interface LanguageConfig {
+  language: string;
+  dockerfilePath: string;
+  runCommand: string;
+}
+
+export const LANGUAGE_CONFIGS: Record<string, LanguageConfig> = {
+  typescript: {
+    language: "typescript",
+    dockerfilePath: path.join(
+      __dirname,
+      "../../supportedLangaugeDockerfiles/typescript/Dockerfile",
```

Comment:
```suggestion
      "../../supportedLanguageDockerfiles/typescript/Dockerfile",
```

Reply:
@daviroo can pinpoint what changed here, can you elaborate?

Reply:
Language is misspelled.

Reply:
Damn, I got a bug here cause of this, I was why the pushed code was not compiling, ha ha, I copied the name in the folder itself and paste it here. Didn't notice it was misspelled. Thanks for the catch, will change the folder name too.